app/assets/javascripts/oxalis/view/right-menu/comment_tab/comment_tab_view.js

```diff
@@ -105,6 +108,11 @@ class CommentTabView extends React.PureComponent<Props, CommentTabStateType> {
     return { data };
   }
 
+  constructor(...args: any) {
```

Comment:
You don't need the constructor for this static assignment, instead you can do this right in line 90, as follows:
`storePropertyUnsubscribers: Array<Function> = [];`